vector/src/main/java/im/vector/riotx/core/platform/EllipsizingTextView.kt

```diff
@@ -0,0 +1,398 @@
+/*
+ * Copyright 2020 New Vector Ltd
```

Comment:
We should keep the original Copyright here, and add NV copyright after

Reply:
Ooops, it has disapeared itself
From this gist: https://gist.github.com/hateum/d2095575b441007d62b8

Reply:
ok, done